flycheck.el

```diff
@@ -6224,6 +6224,9 @@ See Info Node `(elisp)Byte Compilation'."
 (defconst flycheck-emacs-lisp-checkdoc-form
   (flycheck-prepare-emacs-lisp-form
     (require 'checkdoc)
+    ;; elisp-mode provides the elisp syntax table
+    (unless (require 'elisp-mode nil t)
```

Comment:
@cpitclaudel I prefer to use symbols for non-nil arguments, i.e. `(require … … 'no-error)`, because that's easier to read than the meaningless `t` :)

I'll change that before merging :)